### DISCUSSION TITLE
Welcome to FIRE-NFX-V2 Discussions!

### DISCUSSION THREAD

**Discussion by nfxbeats** [nfxbeats/FIRE-NFX-V2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MrBeatmaker223** [comment]
Hi everyone, I'm a beatmaker 30 years,. i make mostly hip hop and Trap beats. Im here to get the most out of my akai fire.

first of all thanks all Mrwarbeats for your script for the AKAI FIRE ! You are a beast. I really love the features in it.

I have a question relating the note repeat function. When I record, for example, hi hat with note repeat in the piano roll, it gives me oversized blocks. It is not useful that way and even quantizing won't help. Is it possible for you to make them short that they fit next to each other according to the Time division?
Is it possible for you to make a note repeat function where you hold for example a note and the switch time divisions on other buttons while still holding that note?

other question; is there a quantize button?

**Comment by nfxbeats** [comment]
1) The note repeat function is not 100% reliable at the moment. I only use it for quick short blast on ocassion. for trap style rolls, you can use other methods like chopping tool or even the built in arpeggiator

2) There is no built in quantizer (under development) but you can set the recording properties to quantize for you. right click the record button in FL to see the options.

**Comment by PacoGuevara95** [comment]


**Reply by nfxbeats** [reply]
the code currently tries to fit the song into the available pads by number of bars. IOW if you had 24 bars in the song that would be 1 per bar when the macro/grid is visible. try turning off the macro/grid buttons to make it so you have 32 pads available and it might fit. Otherwise you can also try to use a selection and have the prgress pads work within only the selection.

**Reply by PacoGuevara95** [reply]
ok this worked and i fixed my problem by changing the number of bars. Nfx is the way to go when using fl studio! congrats!. just one thing you might help me with. since I don't use the Cut copy paste macros all that much i would like to know how to change those macros to save export and redo.
